### PR TITLE
[Translation] Added support for multiple files or directories in XliffLintCommand

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/10368

Same approach of https://github.com/symfony/symfony/pull/28521

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
         "symfony/property-info": "<3.4",
         "symfony/serializer": "<4.1",
         "symfony/stopwatch": "<3.4",
-        "symfony/translation": "<4.2",
+        "symfony/translation": "<=4.2",
```
[the remark this answers is not in the document]
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: We can just revert all the changes for this file (it's okay if the `deps=high` build fails in this case).

[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: > it's okay if the deps=high build fails in this case

Actually no, it's not OK on master: a deps=high failure on master means 4.1 will break with the change when using 4.2 deps.
But yes! the composer.json changes must be reverted.

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/XliffLintCommandTest.php [hunk 2]
```diff
@@ -64,7 +64,7 @@ public function testLintFilesFromBundleDirectory()
     {
         $tester = $this->createCommandTester($this->getKernelAwareApplicationMock());
         $tester->execute(
-            array('filename' => '@AppBundle/Resources'),
+            array('filename' => array('@AppBundle/Resources')),
```
Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/XliffLintCommandTest.php]: although not ideal, using `class_exists(IntlMessageFormatter::class) ? '...' : array('...')` could fix the deps=high build

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/Tests/Command/XliffLintCommandTest.php]: hmm no actually... we would need to patch 4.1

File: src/Symfony/Bundle/FrameworkBundle/composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: Reverted, but `deps=high` will fails, not sure how to solve it :( I tried how far I could, so I need help here.

[the remark this answers is not in the document]
Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: Proposal to patch 4.1 in #28525

**Review by chalasr — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**